src/command/root-command/index.ts

```diff
@@ -95,7 +99,9 @@ export class RootCommand {
       this.maybeSetFromConfig(option)
     })
 
-    this.fdpStorage = new FdpStorage(this.beeApiUrl, this.beeDebugApiUrl)
+    const batchId = this.beePostageBatchId || (await getUsableBatch())
```

Comment:
the problem with this is not all commands require valid postage stamp as some do not upload anything.
either introduce a property in the rootCommand which is a flag for `postageStampRequired` or define postage stamp option on upload commands, but like this is not correctly defined.